pkg/reconciler/common/releases.go

```diff
@@ -85,9 +98,40 @@ func componentDir(instance v1alpha1.KComponent) string {
 	return ""
 }
 
+func componentURL(version string, instance v1alpha1.KComponent) string {
+	component := "serving"
+	urlList := []string{}
```

Comment:
```suggestion
	var urlList []string
```

No need to allocate an object here. I'd also rename this. It doesn't contain a list of URLs but of filenames.